### PR TITLE
fix(datashield): Make ready for demo

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [x] Functionality works & meets specifications (tested on rancher)
- [x] Templates reviewed
- [x] Added values to questions
- [x] Bumped the chart version
- Updated appVersion (if needed)

### CODE REVIEW

File: charts/molgenis-datashield/values.yaml [hunk 1]
```diff
@@ -20,19 +20,18 @@ server:
     #   cpu: 100m
```
Comment (sidohaakma) [inline charts/molgenis-datashield/values.yaml:20]: Bump version of rserver as well. The other pr is merged. There should be an image tag available soon!

**Comment by fdlk** [comment]
I think there's a many to one relationship between a fusion auth server and a datashield instance. each instance is a client id in the fusion auth server

File: charts/molgenis-datashield/values.yaml (hunk 1 above)
Comment (fdlk) [reply charts/molgenis-datashield/values.yaml:20]: *giggles at "soon"* upgrading it

Comment (fdlk) [reply charts/molgenis-datashield/values.yaml:20]: Done

**Pull request merged**